samples/notebooks/python/11-streaming-completions.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
Add a quick preamble for this notebook. "In this notebook we show..."

Reply:
Resolving to merge. We can add detail in a follow-up PR